lib/widgets/message_list.dart

```diff
@@ -781,58 +783,81 @@ class MessageWithPossibleSender extends StatelessWidget {
     final time = _kMessageTimestampFormat
       .format(DateTime.fromMillisecondsSinceEpoch(1000 * message.timestamp));
 
+    final Widget senderWidget;
+    if (item.showSender) {
+      senderWidget = Padding(
```

Comment:
In the end this widget is only partly about the sender, because it also has the timestamp; so `senderWidget` is a bit misleading. A good name might be `senderRow` — it's the row with the sender (which also happens to have a bit more than the sender).